pooch/downloaders.py

```diff
@@ -75,6 +81,7 @@ class HTTPDownloader:  # pylint: disable=too-few-public-methods
 
     def __init__(self, **kwargs):
         self.kwargs = kwargs
+        self.progressbar = self.kwargs.pop("progressbar", False) and has_tqdm
```

Comment:
Since we say in the docstring that kwargs are passed to `requests.get`, it's best if `progressbar=False` is an explicit argument here. It's probably also best to just assign the value to the attribute here `self.progressbar = progressbar` and check if `has_tqdm` in the `__call__` method.

Reply:
Another thing to consider adding here is warning in case `progress==True and has_tqdm==False` so the user knows why their progressbar isn't showing up.